core/src/main/java/jenkins/model/JenkinsLocationConfiguration.java

```diff
@@ -58,9 +58,6 @@ public class JenkinsLocationConfiguration extends GlobalConfiguration implements
     private String adminAddress;
     private String jenkinsUrl;
 
-    // just to suppress warnings
-    private transient String charset, useSsl;
-
```

Comment:
Introduced in https://github.com/jenkinsci/jenkins/commit/526c7880e7a0ce73507f6b91fcb993a6d3e53a59 to not have warnings during deserialization.

Probably reasonable to remove at this point, but only because of age.